### ISSUE TITLE
Simple Drawing Tool

### ISSUE THREAD

**Issue by boXseven** [Linus4/csgoverview#20]
Just be able to draw something on the map.

**Comment by Linus4** [comment]
For now, I recommend [gInk](https://github.com/geovens/gInk) as it is a complete tool to draw on the screen and it integrates nicely with csgoverview.

I'll have to see if it is worth implementing this in csgoverview or if it's better to keep the codebase small.

**Comment by boXseven** [comment]
Yeah, of course I saw that. Just thought that a simple integrated pencil tool wouldn't be bad.
Nothing fancy.
